go/vt/sqlparser/token.go

```diff
@@ -746,6 +749,15 @@ func (tkn *Tokenizer) next() {
 	tkn.Position++
 }
 
+// reset clears any internal state
+func (tkn *Tokenizer) reset() {
+	tkn.posVarIndex = 0
```

Comment:
This shouldn't be reset because a request containing multiple statements will need distinct bind vars.